Network/configure-two-factor-authentication-for-client-vpn.md

```diff
@@ -10,7 +10,7 @@ Creating two factor authentication is very easy to do with Client VPN Service. H
 _WARNING: if you enable AD authentication, all existing users will need to re-download their VPN certificates as their existing certificates will no longer function._
 
 1. Go to Network > VPN from the Control Portal menu.
-  ![](../images/vpn-menu.png)
+  ![](../images/VPN.PNG)
```

Comment:
This image file does not exist. You will have to add it. Also, please use lowercase when naming the image file.